src/components/committee-member/comittee-card.json

```diff
@@ -0,0 +1,36 @@
+{
+  "collectionName": "components_comittee_card_comittee_cards",
```

Comment:
There's a typo in this component name - this is probably fixable with a rename, search&replace plus config import